gym-game2048/gym_game2048/envs/game_2048.py

```diff
@@ -259,5 +246,19 @@ def reset(self):
         self.__total_score = 0
         self.__invalid_count = 0
         self.__total_count = 0
+        self.__num_obstacles = 0
         self.__add_two_or_four()
         self.__add_two_or_four()
+
```

Comment:
Updated this function to return a string that can be stored in the environment info dictionary and printed from the main file